config/krakend/settings/workflow_proxy_settings_deprecated.json

```diff
@@ -5,11 +5,6 @@
             "method": "PUT",
             "url_pattern": "/metadata"
         },
-        {
-            "endpoint": "/api/uniflow/metadata/workflow/{name}/{version}",
-            "method": "GET",
-            "url_pattern": "/metadata/workflow/{name}/{version}"
```

Comment:
this endpoint is included in the Public API proposal document. why was removed?

Reply:
Moved to not deprecated file

Reply:
But there is a different method used.